vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -1012,7 +1012,7 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
 
                 let stripEndingSemicolon (sb : StringBuilder) =
```

Comment:
I really don't know. I only added the len check. The rest of the history is from before F# was open-sourced, so it should be checked on the internal repo.